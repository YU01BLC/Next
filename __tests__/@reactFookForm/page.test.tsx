import { render, screen } from "@testing-library/react";
import Page from "../../app/@reactHookForm/page";

describe("@reactHookForm/Pageコンポーネントscreenshot", () => {
  it("snapshot", () => {
    const { asFragment } = render(<Page />);
    expect(asFragment()).toMatchSnapshot();
  });
});

describe("@reactHookForm/Pageコンポーネント表示確認", () => {
  it("テキスト「react hook form」が表示されること", () => {
    render(<Page />);
    const headElement = screen.getByText("react hook form");
    expect(headElement).toBeInTheDocument();
  });
});
