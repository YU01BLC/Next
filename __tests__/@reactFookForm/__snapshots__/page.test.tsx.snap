// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@reactHookForm/Pageコンポーネントscreenshot snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-12b5rbz"
  >
    react hook form
  </div>
</DocumentFragment>
`;
